src/pages/settings/SettingFormInput.tsx

```diff
@@ -61,9 +64,23 @@ const SettingFormInput: FC<Props> = ({
       <Button appearance="positive" type="submit">
         Save
       </Button>
+      {configField.isUserDefined && (
+        <Button
+          appearance="base"
+          hasIcon
+          className="delete-button"
+          type="button"
+          onClick={() => {
+            onDelete(configField.key);
+          }}
+        >
+          <Icon name="delete"></Icon>
+          <span>Delete</span>
+        </Button>
+      )}
       {canBeReset && (
         <Button
-          className="reset-button"
+          className="reset--button"
```

Comment:
```suggestion
          className="reset-button"
```
I think this change should be reverted to match the scss.